grepdemos/cal.com
# fix: Don't AutoScroll to embed when multiple-duration event's booking page is embedded

## What does this PR do?

Fixes #10590
The issue occurs only for an event with multiple durations enabled

- Added an e2e to verify it is not present
- Added an eslint to by default error on use of scrollIntoView unless acknowledged

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?
1. Create an event with multiple durations enabled
2. Embed that event's booking page.
3. Notice that the scroll automatically occurs to that embed.
